services/visual_recognition/v3.js

```diff
@@ -0,0 +1,348 @@
+          msg.payload='see msg.result.error';
+          node.send(msg); 
+        }
+        else {
+          msg.result = {};
+          msg.result['all'] = body;
+          msg.payload='see msg.result'; // to remove any Buffer that could remains
+          node.send(msg); 
+          node.status({});
+        }
+      } // actionComplete    
+
+      var actionComplete2 = function(err, body, other) {
+
+        if (err != null && body==null)
+        {
+          node.status({fill:'red', shape:'ring', text:'call to watson visual recognition v3 service failed'}); 
+          msg.result = {};
+          msg.result['error_code']= err.code;
+          if (!err.error)
+            msg.result['error']= err.error;
+          console.log('Error:', err.code);
+          node.error('Error code : ' +  err.code);
+          return;
+        }
+        else {
+          msg.result = {};
+          msg.result['all'] = body;
+          msg.payload='see msg.result'; // to remove any Buffer that could remains
+          node.send(msg); 
+          node.status({});
+        }
+      } // actionComplete2
+
+      var actionCompleteDeleteClassifier = function(err, body, other) {
+
+        if (err != null && body==null)
+        {
+          node.status({fill:'red', shape:'ring', text:'call to watson visual recognition v3 service failed'}); 
+          msg.result = {};
+          msg.result['error_code']= err.code;
+          if (!err.error)
+            msg.result['error']= err.error;
+          console.log('Error:', err.code);
+          node.error('Error code : ' +  err.code);
+          return;
+        }
+        else {
+          msg.result = 'Deleted classifier_id';
+          msg.payload='see msg.result';
+          node.send(msg); 
+          node.status({});
+        }
+      } // actionCompleteDeleteClassifier
+      
+      // If the input is an image, need to stream the input in, giving time for the
+      // data to arrive, before invoking the service. 
+      node.status({fill:'blue', shape:'dot', text:'Calling '+ feature + ' ...'});
+      if (imageCheck(msg.payload)) {
+        temp.open({suffix: '.' + fileType(msg.payload).ext}, function (err, info) {
+          if (err) {
+            this.status({fill:'red', shape:'ring', text:'unable to open image stream'});          
+            message ='Node has been unable to open the image stream'; 
+            node.error(message, msg);
+            return;        
+          }  
+          stream_buffer(info.path, msg.payload, function () {
+          params['images_file'] = fs.createReadStream(info.path);
+          if ( msg.params != null && msg.params.classifier_ids != null)
+            params['classifier_ids']=msg.params['classifier_ids'];
+          if ( msg.params != null && msg.params.owners != null)
+            params['owners']=msg.params['owners'];
+          if ( msg.params != null && msg.params.threshold != null)
+            params['threshold']=msg.params['threshold'];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'params'.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2686950655/issues/source?bid=3225115&fileBranchId=3338101#l253)